libc/startup/linux/aarch64/start.cpp

```diff
@@ -223,7 +19,7 @@ extern "C" void _start() {
   // will take us to the previous stack pointer. That is the reason why the
   // actual business logic of the startup code is pushed into a non-inline
   // function do_start so that this function is free of any stack usage.
-  app.args = reinterpret_cast<LIBC_NAMESPACE::Args *>(
+  app.args = reinterpret_cast<Args *>(
```

Comment:
nit: given that the namespace is only used twice, it's probably better to do `LIBC_NAMESPACE::Args` and `LIBC_NAMESPACE::do_start` instead of a `using` at the start